hal/include/sonar.h

```diff
@@ -61,8 +63,10 @@ typedef struct
 	float 	min_distance; 		///< Minimum distance the sensor can measure in centimeters
 	float 	max_distance; 		///< Maximum distance the sensor can measure in centimeters
 	float 	current_distance;	///< Measured distance in meters
+	float	current_velocity;	///< Computed velocity in m/s
 	float 	covariance; 		///< Measurement covariance in centimeters, 0 for unknown / invalid readings
 	bool 	healthy;			///< Indicates whether the current measurement can be trusted
+	bool	healthy_vel;		///< Indicated whether the velocity estimation can be trusted
```

Comment:
It is not "if and only if", the sonar altitude can be healthy when the sonar velocity not. 

As we need at least two consecutive measurements for the velocity, the sonar velocity will be healthy after the second measurement only. Otherwise, we will take a really old value of the sonar to estimate the velocity which will lead to a bad estimation at first.